apps/web/stores/NavigationStore.ts

```diff
@@ -90,44 +121,54 @@ export class NavigationStore {
 
   /*
    * Back button is disabled if:
-   * - current question is the first question
+   * - current question is the first question in the history
    */
   get backButtonIsDisabled() {
```

Comment:
Should this be enabled to allow for users to update the code selections they want to review? 

<img width="1415" alt="image" src="https://github.com/user-attachments/assets/7faba01d-e852-4ba4-a281-76f333c64e40">

Reply:
No, this is what the breadcrumbs would be for. That back only ever operates within a walkthrough, it doesn't navigate to a different page.